src/FINDOLOGIC/Export/Data/Item.php

```diff
@@ -130,6 +143,36 @@ public function addPrice($price, $usergroup = '')
         $this->price->setValue($price, $usergroup);
     }
 
+    public function getInsteadPrice()
+    {
+        return $this->insteadPrice;
+    }
+
+    public function setInsteadPrice($insteadPrice)
+    {
+        $this->insteadPrice = $insteadPrice;
+    }
+
+    public function getMaxPrice()
+    {
+        return $this->maxPrice;
+    }
+
+    public function setMaxPrice($maxPrice)
+    {
+        $this->maxPrice= $maxPrice;
```

Comment:
Space before the equal sign. Odd that this isn't covered by the php-cs-fixer

Reply:
:heav